cli/src/ledgers.rs

```diff
@@ -27,3 +27,38 @@ pub fn all(client: &Client, matches: &ArgMatches) -> Result<()> {
     });
     res
 }
+
+pub fn payments(client: &Client, matches: &ArgMatches) -> Result<()> {
+    let pager = Pager::from_arg(&matches);
+    let sequence = matches
+        .value_of("sequence")
+        .expect("Ledger sequence is required");
+
+    let sequence = sequence.parse::<u32>().unwrap();
+    let endpoint = ledger::Payments::new(sequence);
+    let endpoint = pager.assign(endpoint);
+    let endpoint = cursor::assign_from_arg(matches, endpoint);
+    let endpoint = ordering::assign_from_arg(matches, endpoint);
+
+    let iter = sync::Iter::new(&client, endpoint);
+
+    let mut res = Ok(());
+    pager.paginate(iter, |result| match result {
+        Ok(op) => {
+            println!("ID:           {}", op.id());
+            if let &OperationKind::Payment(ref pymt) = op.kind() {
+                println!("To account:   {}", pymt.to());
+                println!("From account: {}", pymt.from());
+                println!("Asset Type:   {}", pymt.asset().asset_type());
+                println!("Asset Code:   {}", pymt.asset().code());
+                println!("Asset Issuer: {}", pymt.asset().issuer());
+                println!("Amount:       {}", pymt.amount());
+            } else {
+                panic!("Did not receive Payment operation");
```

Comment:
If this ever happens I'll 😱 

Would it matter if we just left this `else` arm off?

Reply:
@kbacha Yes we can get rid of the else arm. @robertDurst very interesting.  I can't seem to get a path_payment to be returned from the api, but i definitely believe that it could be.  Will match on the result to do the printing accordingly.  Thanks for flagging this 👍